tasks/infra_ops.py

```diff
@@ -42,3 +42,12 @@ def load_infrastructure_schema(
         if namespace == Namespace.DEV:
             command_menu = f"{base_cmd} run {SERVICE_WORKER_NAME} infrahubctl menu load {menu_target}"
             execute_command(context=context, command=command_menu)
+
+
+def load_infrastructure_menu(context: Context, database: str, namespace: Namespace) -> None:
```

Comment:
FYI, This command is already part of `load_infrastructure_schema` (just above), it's just that for now it is only applied for the DEV environment